src/cpp/core/include/core/http/SocketAcceptorService.hpp

```diff
@@ -32,6 +32,16 @@ namespace rstudio {
 namespace core {
 namespace http {
 
+namespace {
+
+   inline boost::asio::io_context* newContext()
+   {
+       errno = 0; // Workaround boost::asio bug 1588 - when this is 34 - it throws "config out of range"
```

Comment:
Can you include a link to the bug tracker / issue? I looked in https://github.com/boostorg/asio/issues and those only go up to ~449, and https://github.com/boostorg/boost/issues up to ~1079.

Mostly just wanting to know if this is fixed in a newer boost release as data on whether we consider a boost upgrade in Apple Blossom.